pkgs/top-level/all-packages.nix

```diff
@@ -5800,6 +5800,8 @@ with pkgs;
 
   trash-cli = callPackage ../tools/misc/trash-cli { };
 
+  traverso = libsForQt5.callPackage ../applications/audio/traverso { };
```

Comment:
It should be further down between `transgui` and `trayer`.
Reason is this is an "application" and not a "tool"

Reply:
Ah! I see the "###" section markers now haha